Chap25 (#68)
LaunchCodeEducation/data-analysis: src/chapters/tableau-intro/studio.rst

Studio: Tableau Part 1 - Introduction to Visualization 
======================================================

Getting Started
---------------

> | Please use your best judgement on the type of chart, layout, and interpretation of the list below. This is an exercise to get hands-on experience with Tableau. This will be the first stop on a journey of many skills!

Reviewer: ```suggestion
Please use your best judgement on the type of chart, layout, and interpretation of the list below. This is an exercise to get hands-on experience with Tableau. This will be the first stop on a journey of many skills!
```

> #. We will be using the `Mushroom Classification <https://www.kaggle.com/datasets/uciml/mushroom-classification>`__ dataset from kaggle.  Download the csv file.

Reviewer: ```suggestion
#. We will be using the `Mushroom Classification <https://www.kaggle.com/datasets/uciml/mushroom-classification>`__ dataset from Kaggle.  Download the CSV file.
```

#. Open Tableau Public and connect to your dataset.  

> #. Create visualizations for the following:  (a sheet for each).
> 
>    #. How many Mushrooms are poisonous vs edible?
>    #. How many different habitats were mushrooms collected from?
>    #. How many Mushroom habitats compared to class?
>    #. How many Mushrooms sorted by class and cap-color?
>    #. `Create aliases <https://help.tableau.com/current/pro/desktop/en-us/datafields_fieldproperties_aliases_ex1editing.htm>`__ for each letter in chart d.
>       
>       a. Use the context provided in the dataset for what each letter represents.
>       #. Explore the “Marks” card to add color or size to chart d based on the chart design you have chosen.
>       #. Give your chart a new title.
>       
>    #. Create your own chart with at least 2 dimensions and aliases to provide context.
>    #. Create a dashboard with at least 2 of your sheets.
>    #. Save and publish your dashboard.  

Reviewer: ```suggestion
#. You will need to create visualizations for the following four questions. You only need a sheet for each question.

   #. How many Mushrooms are poisonous vs edible?
   #. How many different habitats were mushrooms collected from?
   #. How many Mushroom habitats compared to class?
   #. How many Mushrooms are sorted by class and cap color?

#. For the fourth question, create aliases <https://help.tableau.com/current/pro/desktop/en-us/datafields_fieldproperties_aliases_ex1editing.htm>`__ for each letter in chart d.
      
      #. Use the dataset's context for each letter's representation.
      #. Explore the “Marks” card to add color or size to chart d based on your chosen chart design.
      #. Give your chart a new title.
      
#. Create your own chart with at least 2 dimensions and aliases to provide context.
#. Create a dashboard with at least 2 of your sheets.
#. Save and publish your dashboard.  
```

Author: This formatting I find confusing.  It breaks up the list of sheets.  It also could cause confusion because you would create a #4 referencing a number 4.  Courtney what is chart d? Is it the chart from #4 above : How many Mushrooms are sorted by class and cap color?

Reviewer: Hey Kimberly! It is just a suggestion! I found the original list extremely confusing because of how many levels of nested lists there are on this page. If you have an alternate idea to break up this list, let me know! If you find this version more confusing and want to move forward, then you can do so.
